### PR TITLE
the *default* branch is called 'main'

### PR DESCRIPTION
**Pull request by woutdenolf**

### CODE REVIEW

**Review by PeterC-DLS — APPROVED** [review]
LGTM

**Comment by prjemian** [comment]
On [GitHub](https://github.com/nexusformat/definitions/settings/branches) (maybe even with git), this is named the *default* branch:  see #884

File: .github/workflows/publish-sphinx.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
 
     steps:
     - name: Checkout
-      uses: actions/checkout@master
+      uses: actions/checkout@main
```
Comment (prjemian) [inline .github/workflows/publish-sphinx.yml:14]: I see, refers to default branch pf this repo: https://github.com/actions/checkout
Thanks.

File: .github/workflows/publish-sphinx.yml [hunk 2]
```diff
@@ -94,15 +94,15 @@ jobs:
         wget https://github.com/nexusformat/code/raw/master/applications/NXtranslate/docs/NXtranslate.pdf -O ./build/manual/source/_static/NXtranslate.pdf
 
     - name: Build (html) and Commit
-      uses: sphinx-notes/pages@master
+      uses: sphinx-notes/pages@main
```
Comment (prjemian) [inline .github/workflows/publish-sphinx.yml:97]: Not seeing a branch named `main` here: https://github.com/sphinx-notes/pages/branches

File: .github/workflows/publish-sphinx.yml [hunk 3]
```diff
@@ -94,15 +94,15 @@ jobs:
...
       with:
         # path to conf.py directory
         documentation_path: build/manual/source
 
     - name: Publish if refs/tags
       # remove/comment next line to push right away
       if: startsWith(github.ref, 'refs/tags')
-      uses: ad-m/github-push-action@master
+      uses: ad-m/github-push-action@main
```
Comment (prjemian) [inline .github/workflows/publish-sphinx.yml:105]: not seeing a branch named `main` here: https://github.com/ad-m/github-push-action/branches

**Comment by woutdenolf** [comment]
yes indeed
```
Error: Unable to resolve action `ad-m/github-push-action@main`, unable to find version `main`. Unable to resolve action `sphinx-notes/pages@main`, unable to find version `main`
```

**Pull request closed without merging**